cmake/DebugRelease/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required(VERSION 3.0)
```

Comment:
We have to increase the cmake version here, just like in the main files!
I totally forgot about that, will fix it in my commit.